tests/test-users-api.js

```diff
@@ -29,20 +29,27 @@ const cmnd = '82764619'
 describe("users api", async() => {
     let new_user_id = null;
     before (async() => {
-        let result = await pool.query('SELECT public."InsertUser"($1, $2, $3, $4, $5, $6, $7) as new_id', 
-            [birthdate, signupdate, name, cmnd, email, gender, expirydate])
-        new_user_id = result.rows[0].new_id
-        console.log('id: ' + new_user_id)
+        // let result = await pool.query('SELECT public."InsertUser"($1, $2, $3, $4, $5, $6, $7) as new_id', 
+        //     [birthdate, signupdate, name, cmnd, email, gender, expirydate])
+        // new_user_id = result.rows[0].new_id
+        // console.log('id: ' + new_user_id)
+        try {
```

Comment:
Why should we need to use try/catch block here?

Reply:
I fixed it